src/main/java/com/dremio/exec/store/hbase/HBaseScanPrel.java

```diff
@@ -59,7 +59,7 @@ public HBaseScanPrel(
       byte[] startRow,
       byte[] stopRow,
       byte[] filter) {
-    super(cluster, traitSet, table, dataset.getStoragePluginId(), dataset, projectedColumns, observedRowcountAdjustment, ImmutableList.of());
+    super(cluster, traitSet, table, dataset.getStoragePluginId(), dataset, projectedColumns, observedRowcountAdjustment, ImmutableList.of(), ImmutableList.of());
```

Comment:
Similar comment as above.

Reply:
Got it. Empty lists are passed here and in the other file because the HBase connector class constructors do not provide such information to be passed to the parent class constructor, in this case a list of `RelHint`'s. I followed suit since this was how the existing code was handling another similar argument passed in the above code. I will make the change and improve the readability.

Reply:
Tried to make the change locally and realized that explicitly declaring the new input will require the addition of the whole calcite core dependency to the hbase connector as RelHint is a class provided by Calcite. Maybe this is why the existing code took the more implicit approach, to simplify the dependencies? Do you think we should include the calcite dependency for this change?

Reply:
All good. Thanks for trying :)